h2o-r/ensemble/h2oEnsemble-package/DESCRIPTION

```diff
@@ -6,7 +6,7 @@ Date: 2017-08-02
 Author: Erin LeDell
 Maintainer: Erin LeDell <erin@h2o.ai>
 Description: H2O Ensemble implements the Super Learner ensemble (stacking) algorithm using the H2O R interface to provide base learning algorithms.   
-License: Apache License (== 2.0)
+License: GNU Affero General Public License (== 3)
```

Comment:
Should also be AGPL-3.
https://github.com/h2oai/h2o-3/pull/15533#discussion_r1210245631

Reply:
Thank you!